src/slicer.cpp

```diff
@@ -173,92 +172,101 @@ bool SlicerLayer::PossibleStitch::operator<(const PossibleStitch &other) const
     return false;
 }
 
-void SlicerLayer::connectOpenPolylinesImpl(Polygons& open_polylines, coord_t max_dist, coord_t cell_size, bool allow_reverse)
+std::priority_queue<SlicerLayer::PossibleStitch>
+SlicerLayer::findPossibleStitches(
+    Polygons& open_polylines, coord_t max_dist, coord_t cell_size, bool allow_reverse)
 {
-    // below code closes smallest gaps first
-
     std::priority_queue<PossibleStitch> stitch_queue;
 
+    int64_t max_dist2 = max_dist * max_dist;
```

Comment:
I usually pass the "-b" option to select the git diff formatting if there are a lot of whitespace changes.  If you know the commit id, then "git show -b 1a8b4c6655af66b9954a96148e2383e673148c7f" shows the diff treating all whitespace sequences with 1 or more spaces as equivalent.  It also works with "git log -p -b" and the regular "git diff" for changes in the local working copy.